ocaml/xapi/xapi_clustering.ml

```diff
@@ -165,8 +165,10 @@ module Daemon = struct
     | None -> ignore (Helpers.call_script script params)
 
   let service = "xapi-clusterd"
+  let xapi_clusterd_port = "8896"
   let enable ~__context =
     debug "Enabling and starting the clustering daemon";
+    maybe_call_script ~__context "/etc/xapi.d/plugins/firewall-port" ["open"; xapi_clusterd_port];
```

Comment:
This path, and the other strings in here, should be defined in xapi_globs.ml, such that they can be set from the config file.

Reply:
@robhoes Just to make sure I understand properly we should move the path, the port number and the "open"/"close" strings to xapi_globs.ml?

And then just call them here with `Xapi_globs.xapi_clusterd_port`?

Reply:
I think that you can leave "open"/"close", and just do the path and port. Grep for e.g. `nbd_firewall_config_script` in xapi_globs.ml to see how to refine them and register them with the config file logic.

Reply:
Should I actually update xapi_conf to include the port? looks like a look of ports are included but commented out. Thats what I've done atm.

Reply:
That's handy yes. How about adding the other one as well?

Reply:
Btw, they are commented out, with the default value filled in, as examples in the config file. That is common practice for such files. So you've done the right thing.

Reply:
I didn't see any of the other paths included in there but I can include the one I've added anyways. Seeing as it is commented out there is no harm.